clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -538,13 +539,20 @@ public ApiException build(String message) {
                 return new UnknownProducerIdException(message);
             }
     }),
-    REASSIGNMENT_IN_PROGRESS(60, "A partition reassignment is in progress",
+    REASSIGNMENT_IN_PROGRESS(60, "A partition reassignment is in progress.",
         new ApiExceptionBuilder() {
             @Override
             public ApiException build(String message) {
                 return new ReassignmentInProgressException(message);
             }
-    });
+    }),
+    INCOMPATIBLE_WITH_CONFIG(61, "The operation requested is incompatible with the configuration.",
```

Comment:
"incompatible with the *broker's* configuration" would be a bit clearer, I think.

Reply:
I removed this error code.